### ISSUE TITLE
influence of diffrent normalization

### ISSUE THREAD

**Issue by hua-wu-que** [autonomousvision/occupancy_networks#62]
Hi there,

I want to ask your advice regarding the normalization layers. I am implementing a network similar to yours, but I found if I use batch norm layer, during testing time model.eval() will give much worse results. Did you encounter this before? Thank you for your reply!

**Comment by AlexsaseXie** [comment]
I did several experiments about BN layers for OccNet. With BN layers, the training will be much stable and the final reconstruction results will be more 'average' and generally well. Without BN layers you cannot easily train the network(the loss fluctuated) and final results are statistically worse.

**Comment by hua-wu-que** [comment]
> I did several experiments about BN layers for OccNet. With BN layers, the training will be much stable and the final reconstruction results will be more 'average' and generally well. Without BN layers you cannot easily train the network(the loss fluctuated) and final results are statistically worse.

Hi AlexsaseXie, did you call model.eval() before doing inference ? I get very bad results if I call model.eval()
